### DISCUSSION TITLE
Any way to automate gas boiler?

### DISCUSSION THREAD

**Discussion by LennardPlay** [jmcollin78/versatile_thermostat#288] [General]
Hey Everyone (I'm new to Github, so please excuse if I post this at the wrong place)

Our apartment has a gas boiler which I can control through a shelly and one radiator per room which I can control with smart thermostat valves. (Obviously the radiators will not heat without hot water from the boiler). I want to be able to adjust the temperature on a room by room basis and depending on that turn the boiler on/off to maximize efficiency.

Is there any way to control the central gas boiler through the versatile_thermostat integration, or should I look into making a complex home assistant automation?

Looking forward to your resposes :) Happy Holidays

**Comment by jmcollin78** [comment]
Hello @LennardPlay,

Have you look into the already open discussions ? I think this question have already been answered the following way:
1. listen to the valve open percent attribute of your VTherms,
2. if one goes over 0 -> switch on the boiler switch,
3. else switch off.

This is not natively taken into account but may be one day

**Comment by LennardPlay** [comment]
Sorry for my late reply.
Thank you :) This helps

**Comment by jmcollin78** [comment]
See https://github.com/jmcollin78/versatile_thermostat/issues/234.

I will certainly do it rapidly.
